cloud/vsphere/provisioner/govmomi/update.go

```diff
@@ -48,12 +50,13 @@ func (vc *Provisioner) Update(cluster *clusterv1.Cluster, machine *clusterv1.Mac
 	}
 
 	if _, err := vsphereutils.GetIP(cluster, machine); err != nil {
+		glog.Info("actuator.Update() - did not find IP, waiting on IP")
```

Comment:
switch to using `glog.V(4).Infof(...` so that we can selectively emit more or less logs